src/main/java/org/isf/exa/manager/ExamBrowsingManager.java

```diff
@@ -131,9 +119,30 @@ public boolean isKeyPresent(Exam exam) throws OHServiceException {
 		return ioOperations.isKeyPresent(exam);
 	}
 
+	/**
+	 * Insert a new {@link Exam} in the DB with exam rows.
```

Comment:
I personally find "in the DB" as being redundant; where else would it be inserted?    There are other examples in this PR like this too.

Reply:
OK ! Let me fix it ! I din't take time to check existing doc.